client/src/components/ButtonWatchlist.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import { ReactComponent as IconMinus } from '../assets/icons/minus-solid.svg'
+import { ReactComponent as IconPlus } from '../assets/icons/plus-solid.svg'
+
+ButtonWatchlist.propTypes = {
+  isOnWatchlist: PropTypes.bool.isRequired,
+  onClick: PropTypes.func.isRequired,
+}
+
+export default function ButtonWatchlist({ isOnWatchlist, onClick }) {
+  return (
+    <StyledButton onClick={onClick} isOnWatchlist={isOnWatchlist}>
+      {isOnWatchlist ? (
+        <>
+          <IconMinus />
+          <div>entfernen</div>
```

Comment:
A `div` element is a `block` element and therefore shouldn't be inside an `inline` element such as the `button`.
If you want a block element here, use a `span` and set it to `display: block;`